test/application_system_test_case.rb

```diff
@@ -1,4 +1,6 @@
-require "test_helper"
+# frozen_string_literal: true
```

Comment:
We should actually remove the `test` directory and all its files, and just keep `spec` since we're going to use RSpec for tests

Reply:
nvm, I saw you're doing this in the other PR